waddrmgr/db.go

```diff
@@ -1641,227 +1665,292 @@ func createManagerNS(namespace walletdb.Namespace) error {
 // upgradeToVersion2 upgrades the database from version 1 to version 2
 // 'usedAddrBucketName' a bucket for storing addrs flagged as marked is
 // initialized and it will be updated on the next rescan.
-func upgradeToVersion2(namespace walletdb.Namespace) error {
-	err := namespace.Update(func(tx walletdb.Tx) error {
-		currentMgrVersion := uint32(2)
-		rootBucket := tx.RootBucket()
+func upgradeToVersion2(tx walletdb.Tx) error {
+	rootBucket := tx.RootBucket()
```

Comment:
Thanks, yeah fixed both issues with rebase. Just moved the definition of `upgradeManager` to the end, after all `upgradeToVersion` funcs.